### ISSUE TITLE
Drag & drop doesn't work in editor

### ISSUE THREAD

**Issue by michalpawl** [Tampermonkey/tampermonkey#1481]
## Expected Behavior
Dragging & dropping selected text moves it to where it's dropped.

## Actual Behavior
When dragging & dropping text inside the script editor, selected text is not moved and a following warning message appears in web console:
```
Error: incompatible TransferableData
    dn moz-extension://3dd42f53-6e49-4b4d-b99d-b614b2b4f7ed/extension.js:332
    ir moz-extension://3dd42f53-6e49-4b4d-b99d-b614b2b4f7ed/extension.js:513
extension.js:513:376
```
Dropping text onto any `textarea` or `input` element in settings also produces the same result.
## Specifications
- Firefox: 99.0 (64-bit)
- TM: 4.16.6160
- OS: Windows 10 version 21H2 build 19044.1586

**Comment by derjanb** [comment]
Should be fixed at 4.17.6161 (in review|[crx](https://tampermonkey.net/crx/tampermonkey_beta_current.crx)|[xpi](https://firefox.tampermonkey.net/firefox-current-beta.xpi))

Chrome/Edge users, please download the crx file linked above and drag and drop it to the extensions page `chrome://extensions`.
Firefox users please install the [BETA version](https://www.tampermonkey.net/index.php?browser=firefox) or check for BETA version updates at `about:addons` 

For a quick fix please export your settings and scripts as zip or (JSON) file at the "Utilities" tab and import it back at the fixed BETA version.

**Comment by michalpawl** [comment]
Works as expected in 4.17.6161
